quipucords/tests/factories.py

```diff
@@ -10,6 +10,7 @@
 from api.status import get_server_id
 from constants import DataSources
 from tests.utils import fake_rhel, raw_facts_generator
+from tests.utils.raw_facts_generator import fake_installed_products
```

Comment:
it's a silly preference of mine and def not required what do you think about having `fake_installed_products` imported on `test.utils.__init__` so it can be imported directly from `test.utils`?

(Its a good moment to discuss this pattern - I will stop doing it if we don't agree)

Reply:
I don't think I have a strong opinion about masking deeper package paths like that. I think the only time I do something like that is when it's technically required, such as when there's an inter-package dependency issue, and exposing deeper imports via a higher package can avoid unexpected import cycles.